CI/windows/build_libvpx.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+################################################################################
+# Windows libvpx cross-compile build script
+################################################################################
+#
+# This script file can be included in build scripts for Windows or run directly
+#
+################################################################################
+
+# Halt on errors
+set -eE
+
+_fixup_libs() {
+    vpxname=`find . -type f -iname libvpx*.dll`
```

Comment:
Use `UPPERCASE` for variables, use `$()` for subcommand invocations.

On second look I'd use `VPXNAME="$(find . -type f -iname "libvpx*.dll" | xargs basename)"`.

Be aware that `find` could yield multiple results, which `xargs` will work with, but your variable will contain multiple values. `head -n1` might be necessary then.

Reply:
This might be even better as it will invoke it for every file it finds:

```
find . -type f -iname "libvpx*.dll" | xargs -I{} sh -c "${WIN_CROSS_TOOL_PREFIX}-w64-mingw32-dlltool -m ${WIN_CROSS_MVAL} -d libvpx.def -l \"${BUILD_DIR}/bin/vpx.lib\" -D \"${BUILD_DIR}/bin/{}\""
```

Reply:
> Be aware that `find` could yield multiple results

In a clean build directory, there _should_ be only one result for this.  The only reason for the wildcard is to account for the version suffix (in this case, "-1").  We could probably detect what the suffix should be (e.g., from `build/make/version.sh` or the git tag version), and therefore the full output filename, instead of messing with find.  I just didn't do that for this implementation because what I had at the moment worked.

> `head -n1` might be necessary then.

If there were multiple results, would this return the latest/newest libvpx?  Would that be desirable?

Reply:
Not the newest, just the first. You'd probably have to adjust the find parameters, but you can optimise this later.